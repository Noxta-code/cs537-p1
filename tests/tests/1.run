../solution/sortk tests/1.input 1
