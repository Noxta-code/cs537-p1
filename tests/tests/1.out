and great achilles first fell out with one another
and which of the gods was it that set them on to quarrel
for so were the counsels of jove fulfilled from the day on which the son of atreus king of men
many a brave soul did it send hurrying down to hades
many a hero did it yield a prey to dogs and vultures
sing o goddess the anger of achilles son of peleus that brought countless ills upon the achaeans
